src/scrabble.js

```diff
@@ -1,5 +1,127 @@
-function scrabble() {
+function scrabble(string) {
+  // let score = 0
 
+  if (!string) {
+    return 0
+  }
+
+  const word = string.toUpperCase()
+
+  const scoreOfOne = ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T']
+  const scoreOfTwo = ['D', 'G']
+  const scoreOfThree = ['B', 'C', 'M', 'P']
+  const scoreOfFour = ['F', 'H', 'V', 'W', 'Y']
+  const scoreOfFive = 'K'
+  const scoreOfEight = ['J', 'X']
+  const scoreOfTen = ['Q', 'Z']
+
+  function calculateScore(word) {
+    let score = 0
+    for (let i = 0; i < word.length; i++) {
+      const counter = i
+
+      for (let j = 0; j < scoreOfOne.length; j++) {
```

Comment:
Interesting take to run a loop within a loop to check if the letter is in the array. There is a built in function though that makes it a lot easier which is array.includes()